src/common/snippets/tests/src/pass/softmax_reshape_elimination.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/function.hpp>
+#include <ngraph/pass/manager.hpp>
+
+#include <snippets/snippets_isa.hpp>
+#include <snippets/pass/softmax_reshape_elimination.hpp>
+
+#include <transformations/init_node_info.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+using namespace ngraph;
+
+TEST(TransformationTests, SoftmaxV1ReshapeElimination) {
```

Comment:
I suppose we can reuse `TransformationTestsF` class here. [Example](https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/tests/prelu_fusion.cpp#L21)

Reply:
I don't know about these tests. Thank you very much! Updated